File: go/balance.go
// Example showing how to get account balance
package main

import (
	"fmt"
	"time"
	"binary_options_tools_uni"
)

func main() {
	client, err := binary_options_tools_uni.NewPocketOption("your-session-id")
	if err != nil {
		panic(err)
	}
	time.Sleep(5 * time.Second)
	
	balance, err := client.Balance()
	if err != nil {
		panic(err)
	}
	fmt.Printf("Your current balance is: $%.2f\n", balance)
}

File: rust/balance.rs
// Example showing how to get account balance
use binary_options_tools::PocketOption;
use std::time::Duration;

#[tokio::main]
async fn main() -> Result<(), Box<dyn std::error::Error>> {
    // Initialize client
    let client = PocketOption::new("your-session-id").await?;
    
    // IMPORTANT: Wait for connection to establish
    tokio::time::sleep(Duration::from_secs(5)).await;
    
    // Get current balance
    let balance = client.balance().await;
    println!("Your current balance is: ${:.2}", balance);
    
    Ok(())
}
